lib/node/types.mli

```diff
@@ -59,5 +59,5 @@ type 'a node = {
   (* Dissemination component ; runs automatically with the server and is responsible
      for automatically disseminating both new Post messages and received Post messages
      with other nodes in the network *)
-  disseminator : Disseminator.t;
+  mutable disseminator : Disseminator.t;
```

Comment:
What are the advantages to make it mutable?

Reply:
If the fields of the disseminator are not mutable, then the disseminator needs to be mutable. Ultimately because the disseminator maintains state: the messages that are currently being disseminated and how many rounds are left before they're no longer hot enough to disseminate.